faraday/proxy.py

```diff
@@ -581,7 +581,12 @@ def main():
 
     # Associate serial ports with callsigns
     # global units
-    units = callsign2COM()
+    try:
+        units = callsign2COM()
+
```

Comment:
Style nitpick: Don't put a space between the `try` and `catch` blocks since they are logically connected.